lib/createInlinePluginCreator.js

```diff
@@ -226,7 +229,21 @@ function createInlinePluginCreator(packages, multiContext, synchronizer, flags)
 			getLucky("_readyForTagging", pkg);
 			await waitFor("_readyForTagging", pkg);
 
-			updateManifestDeps(pkg);
+			// Get all manifests that potentially needs to be updated
+			const pkgs = getPkgRootOptions(context).map((pkgRoot) => {
+				const manifestPath = cleanPath(`${pkgRoot}/package.json`, context.cwd);
+				return {
+					path: manifestPath,
+					manifest: getManifest(manifestPath),
+					_depsChanged: pkg._depsChanged,
+				};
+			});
```

Comment:
Nice. But let's simplify a bit.

```js
pkgs.includes((item) => item.path === pkg.path) {}
```

Or maybe it's reasonable to dedupe `getPkgRootOptions` result to.

``` 
const pkgRoots = getPkgRootOptions(...)
const pkgs = uniqBy([pkg, ...pkgRoots], 'path') // `uniq` from lodash
```

Reply:
The rest LGTM. Great job on the tests!

Reply:
Done!

Thanks for the tip, I was not aware of `uniqBy` and this was exactly what I wanted to achieve 😉